### ISSUE TITLE
MinItemsError rendering incorrectly

### ISSUE THREAD

**Issue by xjamundx** [sagold/json-schema-library#56]
I'm seeing in practice the too few items error rendering incorrectly as:

**Too few items in `#/separatePolicies/linkFormats`, should be at least `undefined`, but got `1`**

When I look at the code I believe it's because we're trying to interpolate the `minimum` key, but we should be referencing `minItems`. Also the test doesn't actually test that the string is correct it only tests that it's rendering roughly the correct message.

Here's how I'm defining the json schema for that field:

```json
 "linkFormats": {
    "minItems": 3,
    "type": "array",
    "items": {
      "type": "string"
    }
  }
```

If you want I can try to submit a CR for this unless you're very eager to fix.

![Screenshot 2024-03-27 at 11 08 55 am](https://github.com/sagold/json-schema-library/assets/246143/c6645616-c266-46c1-a1d4-58c90c2f5b7b)

**Comment by sagold** [comment]
Happy to receive a fix. Note that you can customize this locally, too.

**Comment by sagold** [comment]
The issues has been fixed with `json-schema-library@9.3.3`
